client/components/Review.jsx

```diff
@@ -7,11 +7,24 @@ class Review extends React.Component {
     super(props);
     this.state = {
       username: '',
-      url: ''
+      url: '',
+      date: ''
     };
   }
 
   componentDidMount() {
+    const date = (this.props.info.data.attributes.date).replace(/\//g,'');
+    fetch(
+      `http://localhost:3004/${date}/timeago`
+    )
+      .then(res => {
+        return res.json();
+      })
+      .then(timeAgo => {
+        this.setState({
+          date: timeAgo
+        });
+      });
```

Comment:
You may want to consider `.catch` blocks in your `fetch` calls with some error statements to help better identify errors.